test/services/api.unit.test.js

```diff
@@ -112,5 +112,20 @@ describe('services > API', () => {
       expect(superagent.set).toHaveBeenCalledWith('Authorization', 'Bearer THE-TOKEN');
       expect(superagent.send).toHaveBeenCalledWith();
     });
+    // eslint-disable-next-line jest/prefer-expect-assertions
```

Comment:
do we need this line?

Reply:
This is debatable:
I dont like specifiying `expect.assertions(7);`, but this is a linting rule on the repo so indeed I need this comment

Reply:
We should follow the repository conventions I think :/

Reply:
Done